src/overlays/actors/ovl_En_Dai/z_en_dai.c

```diff
@@ -26,62 +49,701 @@ const ActorInit En_Dai_InitVars = {
     (ActorFunc)EnDai_Draw,
 };
 
-#endif
-
-extern UNK_TYPE D_060002E8;
-extern UNK_TYPE D_0600C538;
-extern UNK_TYPE D_060130D0;
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dai/func_80B3DFF0.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dai/func_80B3E168.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dai/func_80B3E460.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dai/func_80B3E5B4.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dai/func_80B3E5DC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dai/func_80B3E69C.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dai/func_80B3E7C8.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dai/func_80B3E834.s")
+static Vec3f D_80B3FBF0 = { 1.0f, 1.0f, 1.0f };
+
+EnDaiParticle* func_80B3DFF0(EnDaiParticle* particle, Vec3f arg1, Vec3f arg2, Vec3f arg3, f32 arg4, f32 arg5,
```

Comment:
Since this is an array of particles this should be named `particles` imo.
The same logic applies to the other functions.
```suggestion
EnDaiParticle* func_80B3DFF0(EnDaiParticle* particles, Vec3f arg1, Vec3f arg2, Vec3f arg3, f32 arg4, f32 arg5,
```